doc/devel/document.rst

```diff
@@ -1106,6 +1106,15 @@ minimal amount of code necessary to showcase the feature. The lack of extraneous
 makes it easier for the reader to map which parts of code correspond to which parts of
 the plot.
 
+Figure size
+^^^^^^^^^^^
+The default figure size is suitable in many cases. If you want to use an explicit
+figure size, it should be chosen so that the figure is not wider than the available
```

Comment:
I'd say "it should usually be chosen...".   Sometimes downscaling is fine and it can save having to resize all the elements.  

Also px is a fragile measure and possibly will lead to confusion when the doc build produces images that are 1440 px wide. How is the width of pydata sphinx theme specified?

Reply:
My understanding is that in browser-land "px" is always virtual screen pixels that are mostly stable and the what you want so hidpi is handled.

As much as I love details, maybe something like "do not go above 7.2in wide, which is the maximum size that will not be down-scaled in the rendered html documentation, without good reason"

Reply:
I've tried to condense the text, but feel the additional information is valuable
(i) reason for current numbers = pydata-sphinx-theme - the numbers may become incorrect if pydata-sphinx-theme changes, and it's easier to understand/update when we know where they came from
(ii) give px and `figsize` - I believe very few people think in inches nowadays.
(iii) give the "extended" width when there is not right-hand side menu - this can help people who actually need more space to still fit without downsampling.